provisioning/roles/early_packages/tasks/main.yml

```diff
@@ -1,4 +1,12 @@
 ---
+- name: Install python3-libdnf5
+  # python3-libdnf5 is needed by ansible package module.
+  # It is not installed in Fedora 41 by default.
```

Comment:
Please add another comment to this that says you can't use the packages module without installing this in this manner first.  Because otherwise, one might think, why not just install python3-libdnf5 using the packages module?